src/Functions/Security/Get-RsCatalogItemRole.ps1

```diff
@@ -0,0 +1,120 @@
+            This script retrieves access to SQL Server Reporting Services Instance from users/groups.
+        
+        .DESCRIPTION
+            This script retrieves all access on the SQL Server Reporting Services Instance located at the specified Report Server URI from the specified user/group.
+        
+        .PARAMETER Identity
+            Specify the user or group name to retrieve access for.
+
+        .PARAMETER Path
+            Specify the path to catalog item on the server.
+
+       .PARAMETER Recurse
+            Recursively list subfolders with content.
+        
+        .PARAMETER ReportServerUri
+            Specify the Report Server URL to your SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER Credential
+            Specify the password to use when connecting to your SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER Proxy
+            Report server proxy to use.
+            Use "New-RsWebServiceProxy" to generate a proxy object for reuse.
+            Useful when repeatedly having to connect to multiple different Report Server.
+        
+        .EXAMPLE
+            Get-RsCatalogItemRole -Identity 'jmcgee'
+            Description
+            -----------
+            This command will establish a connection to the Report Server located at http://localhost/reportserver using current user's credentials and then retrieves all access for user 'jmcgee'.
+        
+        .EXAMPLE
+            Get-RsCatalogItemRole -ReportServerUri 'http://localhost/reportserver_sql2012' -Identity 'jmcgee'
+            Description
+            -----------
+            This command will establish a connection to the Report Server located at http://localhost/reportserver_2012 using current user's credentials and then retrieves all access for user 'jmcgee'.
+
+        .EXAMPLE
+            Get-RsCatalogItemRole -ReportServerUri 'http://localhost/reportserver_sql2012' -path '/Path/Human Resources/'
+            Description
+            -----------
+            This command will establish a connection to the Report Server located at http://localhost/reportserver_2012 using current user's credentials and then retrieves all access on catalog items found at '/Path/Human Resources/'.
+        
+        .EXAMPLE
+            Get-RsCatalogItemRole -Credential 'CaptainAwesome' -Identity 'jmcgee' -Recurse
+            Description
+            -----------
+            This command will establish a connection to the Report Server located at http://localhost/reportserver using CaptainAwesome's credentials and then retrieves all access for user 'jmcgee' recursively.
+    #>
+
+    [CmdletBinding()]
+    param
+    (
+        [Alias('UserOrGroupName')]
+        [string]
+        $Identity,
+        
+        [string]
+        $RoleName,
```

Comment:
It seems like this parameter is not being used. If it's not needed, could you remove it?